### DISCUSSION TITLE
Did you consider adding the ability to automatically create keyframes ？

### DISCUSSION THREAD

**Discussion by Secular2020** [mifi/lossless-cut#1627] [Ideas]
When cutting part of the video, I have encountered the problem that the original keyframe cannot be cut due to demand, if it is the tail end, it does not matter, if it is in the middle of the video, this non-keyframe cutting and export will appear the picture stuttering, so I have this recommendation. When cutting video non-keyframes, you can create keyframes autonomously.

**Comment by mifi** [comment]
how to create keyframes? I believe tihs is what the smart cut feature is doing
